frontend/src/pages/projects/screens/detail/connections/ManageConnectionsModal.tsx

```diff
@@ -40,36 +41,38 @@ export const ManageConnectionModal: React.FC<Props> = ({
   const [isModified, setIsModified] = React.useState(false);
 
   const enabledConnectionTypes = React.useMemo(
-    () =>
-      connectionTypes.filter((t) => t.metadata.annotations?.['opendatahub.io/enabled'] === 'true'),
+    () => filterEnabledConnectionTypes(connectionTypes),
     [connectionTypes],
   );
 
+  const connectionTypeRef = connection?.metadata.annotations['opendatahub.io/connection-type'];
```

Comment:
Was this a ref before? should the name be updated?

Other than that, everything look good

Reply:
It was not a react ref. I have renamed it to not be confused with React ref.